### PR TITLE
Homework 2. Parking lot. OOP Project

### PR DESCRIPTION
**Pull request by marjana07**

### CODE REVIEW

File: src/main/java/ParkingLot/Admin.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.company;
```
Comment (olobanok) [inline src/main/java/ParkingLot/Admin.java]: how come you have package name com.company here?
this class is in the ParkingLot package

File: src/main/java/ParkingLot/Admin.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        Standard();
+    }
+
+    public Admin(String name, int workHours) {
+        super(name, workHours);
+        Standard();
+    }
+
+    @Override
+    public void Standard() {
```
Comment (olobanok) [inline src/main/java/ParkingLot/Admin.java]: method name should start with lowercase

File: src/main/java/ParkingLot/Vehicle.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    @Override
+    public String toString() {
+        return this.type + " number: " + this.number + ". Owner: " + this.owner + ". Check in: " + this.getDateIn(); //
+    }
+
+    public boolean checkIn() {
+        if (Main.freeLots >= 0) {
+            Date dateIn = new Date();
+            Main.freeLots = Main.freeLots - 1;
```
Comment (olobanok) [inline src/main/java/ParkingLot/Vehicle.java]: change if-condition to Main.freeLots > 0 as now vehicle may check in even if there are 0 free lots

File: src/main/java/ParkingLot/Admin.java (hunk 1 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Admin.java]: done

File: src/main/java/ParkingLot/Admin.java (hunk 2 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Admin.java]: fixed

File: src/main/java/ParkingLot/Vehicle.java (hunk 3 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Vehicle.java]: done

File: src/main/java/ParkingLot/Admin.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package ParkingLot;
+
+public class Admin extends Employees {
```
Comment (AqaCoreSchool) [inline src/main/java/ParkingLot/Admin.java:3]: Employees is good name for List<Employee> variable, not for a class

File: src/main/java/ParkingLot/Admin.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        standard();
+    }
+
+    public Admin(String name, int workHours) {
+        super(name, workHours);
+        standard();
+    }
+
+    @Override
+    public void standard() {
```
Comment (AqaCoreSchool) [inline src/main/java/ParkingLot/Admin.java:15]: naming convention violation. method names should be verbs, and should describe its behavior

File: src/main/java/ParkingLot/Employees.java [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        this.position = position;
+    }
+
+    public int getWorkHours() {
+        return this.workHours;
+    }
+
+    @Override
+    public String toString() {
+        return this.name + " " + position + ". Salary: " + this.getSalary();
```
Comment (AqaCoreSchool) [inline src/main/java/ParkingLot/Employees.java:51]: just a suggestion you could use a String.format or joiner here

File: src/main/java/ParkingLot/Main.java [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+import java.util.Scanner;
+
+import static ParkingLot.VehicleParking.*;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Employees s1 = new SecurityStaff();
```
Comment (AqaCoreSchool) [inline src/main/java/ParkingLot/Main.java:11]: s1 is a bad name for variable

File: src/main/java/ParkingLot/VehicleParking.java [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+            if (vehicle.getNumber().substring(0, 2).equals(template)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    static void waitConsole() {
+        try {
+            Thread.sleep(1000);
```
Comment (nmelnyk-corevalue) [inline src/main/java/ParkingLot/VehicleParking.java:40]: i'm wondering, what is the motivation for thread sleep?

Comment (marjana07) [reply src/main/java/ParkingLot/VehicleParking.java:40]: if i use it, i can have different checkIN time for Vehicle objects, and sort them by duration.

File: src/main/java/ParkingLot/Admin.java (hunk 4 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Admin.java:3]: fixed

File: src/main/java/ParkingLot/Admin.java (hunk 5 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Admin.java:15]: fixed, changes pushed to mmykytovych_master

File: src/main/java/ParkingLot/Employees.java (hunk 6 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Employees.java:51]: fixed, changes pushed to mmykytovych_master

File: src/main/java/ParkingLot/Main.java (hunk 7 above)
Comment (marjana07) [reply src/main/java/ParkingLot/Main.java:11]: fixed, changes pushed to mmykytovych_master

**Pull request closed without merging**